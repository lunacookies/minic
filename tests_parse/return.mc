func lonely {
	return 10
}

func variable {
	var x = 1
	return x
}

func complex {
	var x = 10
	return 10 * (x - 5)
}
