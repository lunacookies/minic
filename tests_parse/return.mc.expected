func lonely {
	return 10
}

func variable {
	x := 1
	return x
}

func complex {
	x := 10
	return (10 * (x - 5))
}
